src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -266,7 +266,7 @@ class RelationalDBInterfaceSqliteImp : public RelationalDBInterfaceSqlite
     bool
     existsLedger()
     {
-        return (bool)lgrdb_;
+        return static_cast<bool>(lgrdb_);
```

Comment:
https://www.cplusplus.com/reference/memory/unique_ptr/operator%20bool/
The bool() operator in that object is called that does exactly what you want.